fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -147,7 +149,10 @@ void writeResponse(final FcrepoResponse response, final File file)
 
         final List<URI> describedby = response.getLinkHeaders("describedby");
         for (final Iterator<URI> it = describedby.iterator(); describedby != null && it.hasNext(); ) {
-            exportDescription(it.next());
+            exportDescription(response.getUrl(), it.next());
+            if (it.hasNext()) {
+                throw new RuntimeException("fcrepo does not produce multiple describedby headers.");
```

Comment:
That's always the idea, but if it adds complexity (and it does in this case) and adds more unit tests (that cover behavior that fedora does not exhibit) every time we refactor those tests, change to new test libraries, run the build routine, etc. costs us time.  All for the unlikely potential that if/when we change the fedora spec and implementation we *might* not have to change this code as much?  Give me a break.  This code doesn't even do the things we need it to do now... maintaining code to allow it to do something that we couldn't exercise until the spec is complete and the implementation rewritten (or a new one made) seems a pretty pooruse of resources.